jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiController.java

```diff
@@ -137,12 +153,18 @@ private T readEntityFromPayload(String body) throws JsonProcessingException {
     }
   }
 
-  private List<T> readEntitiesFromPayload(String body) throws JsonProcessingException {
+  private List<PatchOperation<T>> readPatchOperationsFromPayload(List<Object> body) {
     try {
-      return objectMapper.readerForListOf(entityType).readValue(body);
-    } catch (UnrecognizedPropertyException ex) {
-      throw new UnknownResourcePropertyException(
-          ex.getPropertyName(), ex.getReferringClass().getSimpleName());
+      var opList = new ArrayList<PatchOperation<T>>();
+      var patchOperationType = objectMapper.getTypeFactory().constructParametricType(
+          PatchOperation.class,
+          entityType);
+
+      body.forEach(ob -> opList.add(objectMapper.convertValue(ob, patchOperationType)));
+
+      return opList;
+    } catch (IllegalArgumentException ex) {
```

Comment:
Is there even any IllegalArgumentException to catch here?

Reply:
[This test](https://github.com/UKHomeOffice/callisto-jparest/pull/59/files#diff-fe8094a534e6ffd393deb6a234f961ffdf807e844e48c4e88302b30310740278R663) manages to reach that code, but I can't recreate that from calling the endpoint with postman, so maybe it isn't necessary? I'll remove it, but let me know if you don't think that's the right way to go

Reply:
Yeah doesn't sounds necessary, think the object mapper just throws it in odd situations. But probs not something for us to need to handle.